app/helpers/service_provider_helper.rb

```diff
@@ -1,4 +1,4 @@
-#:reek:UtilityFunction
+#:reek:UtilityFunction, :reek:TooManyConstants
```

Comment:
note to self, time to just get rid of reek

Reply:
😍